shared/constants/types/tracker2.tsx

```diff
@@ -94,6 +94,11 @@ export type WebOfTrustEntry = {
   vouchedAt: number
 }
 
+export const showableWotEntry = (entry: WebOfTrustEntry): boolean => {
+  // when suggestions are implemented, we'll also want to show rejected entries that have a suggestion
+  return entry.status in [RPCTypes.WotStatusType.accepted, RPCTypes.WotStatusType.proposed]
+}
+
```

Comment:
1. We tend not to put helper functions in the `types` files. I would suggest moving it to `constants/tracker2.tsx`

2. Can do an inline return as an arrow function here. Unless you're certain that there's more logic to be added to this function later.

```suggestion
// when suggestions are implemented, we'll also want to show rejected entries that have a suggestion
export const showableWotEntry = (entry: WebOfTrustEntry): boolean => 
    entry.status in [RPCTypes.WotStatusType.accepted, RPCTypes.WotStatusType.proposed]


```

Reply:
95% certain. i can make it a oneline banger for now though.